android/src/main/java/com/zmxv/RNSound/RNSoundModule.java

```diff
@@ -56,6 +66,19 @@ protected MediaPlayer createMediaPlayer(final String fileName) {
     if (res != 0) {
       return MediaPlayer.create(this.context, res);
     }
+    if(fileName.startsWith("http://") || fileName.startsWith("https://")) {
+      MediaPlayer mediaPlayer = new MediaPlayer();
+      mediaPlayer.setAudioStreamType(AudioManager.STREAM_MUSIC);
+      Log.i("RNSoundModule", fileName);
+      try {
+        mediaPlayer.setDataSource(fileName);
+      } catch(IOException e) {
+        Log.e("RNSoundModule", "Exception", e);
+        return null;
```

Comment:
it usually throws an exception when internet is missing, or when the file does not exist. Do you suggest to handle differently those two? (returning null here, will invoke the error callback anyway later on with a more generic "Resouce not found" error)